testbench.txt

```diff
@@ -0,0 +1,79 @@
+nest_base
+botnet50ts_256
+cait_s24_224
+coat_small
+coatnet_3_224
+convit_base
+cspresnet50
+deit3_medium_patch16_224
+densenet169
+dla102
+dpn107
+edgenext_base
+efficientnet_b4
+efficientnet_es
+efficientnetv2_m
+fbnetc_100
+mixnet_l
+mnasnet_b1
+mnasnet_a1
+mnasnet_small
+mobilenetv2_140
+spnasnet_100
+tinynet_e
+eva_large_patch14_196
+eva02_base_patch14_224
+flexivit_base
+focalnet_base_lrf
+gcvit_base
+ghostnet_130
+gmlp_b16_224
+gernet_m
+halonet50ts
+hrnet_w40
+inception_v3
+inception_resnet_v2
+inception_v4
+lambda_resnet50ts
+levit_384
+maxvit_base_tf_224
+mixer_l16_224
+mobilenetv3_large_100
+fbnetv3_g
+hardcorenas_d
+lcnet_150
+mobilevitv2_150
+mvitv2_base
+nasnetalarge
+nfnet_f3
+nf_regnet_b5
+nf_resnet101
+pnasnet5large
+poolformer_s36
+pit_b_224
+pvt_v2_b4
+regnetz_040
+repvgg_b3
+resmlp_36_224
+resnet152
+resnext101_32x8d
+ecaresnet101d
+ecaresnext50t_32x4d
+seresnet152
+resnetrs200
+res2net101d
+resnest200e
+rexnet_200
+SelecSls84
+skresnet50
+sequencer2d_m
+swin_s3_base_224
+tnt_b_patch16_224
+tresnet_l
+twins_svt_base
+visformer_small
+volo_d3_224
+vovnet57a
+legacy_xception
+xception71
+xcit_medium_24_p16_224
```

Comment:
Of the original full_modellist file, are all these models guaranteed working with the new bs 32?

Reply:
Yes, they are. I've run them and had their log uploaded